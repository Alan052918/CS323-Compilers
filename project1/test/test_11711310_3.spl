float foo()
{
  float f = 1.00000;
  f = f + 1;
  return f;


