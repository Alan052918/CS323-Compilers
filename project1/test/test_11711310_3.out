Error type B at Line 8: Missing closing curly bracket '}'
